flight/Modules/RadioComBridge/RadioComBridge.c

```diff
@@ -379,10 +378,10 @@ static void radioRxTask( __attribute__ ((unused))
 #ifdef PIOS_INCLUDE_WDG
 		PIOS_WDG_UpdateFlag(PIOS_WDG_RADIORX);
 #endif
-		if (PIOS_COM_RADIO) {
+		if (PIOS_COM_RFM22B) {
```

Comment:
To be honest, I would love to see all these magic constants for port IDs vanish and be passed as variable to the init method. Might be appropriate for this refactoring.